releases.go

```diff
@@ -68,9 +72,19 @@ func (q ReleasesQuery) Scope(db *gorm.DB) *gorm.DB {
 		scope = append(scope, FieldEquals("version", *version))
 	}
 
+	if rangeHeader := q.Range; rangeHeader != nil {
```

Comment:
This looks pretty reusable. We could add a factory method that takes a Range and returns a new Scope:

```go
func Range(r *headeruitl.Range) Scope {
        // Set limit
        // Set sort
        // Set offset
}
```

And then use it like:

```go
if q.Range != nil {
        scope = append(scope, Range(q.Range))
}
```